### PR TITLE
OEUI-69: Consolidating Application's CSS

### PR DESCRIPTION
**Pull request by zeze111**
[OEUI-69: Consolidating Application's CSS](https://issues.openmrs.org/browse/OEUI-69)

### SUMMARY: 

This involves adding all the css into `openmrs-owa-orderentry.css` file since it is the only CSS file being bundled by the webpack. With this, the `index.html` will not have to be updated every time some styling is needed.

### CODE REVIEW

**Comment by Betty-Kebenei** [comment]
@zeze111  You have failing tests

File: app/js/components/orderEntry/PatientDashboard.jsx [hunk 1]
```diff
@@ -1,62 +0,0 @@
-import React from 'react';
```
Comment (kodero) [inline app/js/components/orderEntry/PatientDashboard.jsx]: Not sure what am missing here. If we are consolidating `CSS` why were non-css related code affected in this?

File: app/css/openmrs-owa-orderentry.css [hunk 2]
```diff
@@ -77,3 +80,128 @@ body {
 .w-44-percent {
     width: 44%
 }
+/* *
+ * Css for Headetr Component
```
Comment (fuchodeveloper) [inline app/css/openmrs-owa-orderentry.css]: A little type here `Headetr`

**Comment by dkayiwa** [comment]
Can you squash the commits into one?

File: tests/components/orderentry/PatientDashboard.test.jsx [hunk 3]
```diff
@@ -1,10 +0,0 @@
-import React from 'react';
-
-import PatientDashboard from '../../../app/js/components/orderEntry/PatientDashboard';
-
```
Comment (dkayiwa) [inline tests/components/orderentry/PatientDashboard.test.jsx]: Was removing this part of the work for this ticket?

File: app/js/components/orderEntry/PatientDashboard.jsx (hunk 1 above)
Comment (dkayiwa) [reply app/js/components/orderEntry/PatientDashboard.jsx]: @zeze111 that would call for a separate ticket and pull request.

File: tests/components/orderentry/PatientDashboard.test.jsx (hunk 3 above)
Comment (zeze111) [reply tests/components/orderentry/PatientDashboard.test.jsx]: I believe so, from my understanding with the team, I was to remove unwanted and duplicated CSS from our files. The main aim although was putting all the external css into the `opemrs-owa-orderentry.css` file.

File: app/js/components/orderEntry/PatientDashboard.jsx (hunk 1 above)
Comment (zeze111) [reply app/js/components/orderEntry/PatientDashboard.jsx]: okay

**Pull request merged**